commit 7dc7d5abe26a30e59c96b82e6bc44603b9813e89
Author: hazen2215 <haz@disroot.org>
Date:   Sun Mar 16 22:01:23 2025 +0900

    Revert "Add workaround for NetBSD-i386-9.1 with gcc-7.5.0 and -O2"
    
    This reverts commit 7fa131e8234b6e69f67f91e33f16fce9d74e8d1c.

    See: https://github.com/uim/uim/issues/207

diff --git a/configure.ac b/configure.ac
index 282b1c61..379a195c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -425,10 +425,6 @@ AX_CFLAGS_GCC_OPTION([-Wno-overlength-strings])
 # Disable the problematic preprocessing on Mac OS X
 AX_CFLAGS_GCC_OPTION([-no-cpp-precomp])
 
-# Workaround for NetBSD-i386-9.1 with gcc-7.5.0 and -O2.
-# See also: https://github.com/uim/uim/issues/140#issuecomment-752380919
-AX_CFLAGS_GCC_OPTION([-fno-inline-small-functions])
-
 # Deprecated autoconf macro used in AX_CXXFLAGS_GCC_OPTION.
 AC_DEFUN([AC_LANG_CXX], [AC_LANG([C++])])
 
